source/working_with_kayobe.rst

```diff
@@ -62,6 +62,14 @@ After making the checkout, source the virtualenv and Kayobe config environment v
    kayobe# source venvs/kayobe/bin/activate
    kayobe# source src/kayobe-config/kayobe-env
 
+If you are using a Kayobe environment, you will instead need to specify which
+environment to source. See the section Kayobe Environments for more details.
```

Comment:
Nit: Would be nice to use an anchor and link to the section

Reply:
Done :+1: